### DISCUSSION TITLE
0524 피어세션일지

### DISCUSSION THREAD

**Discussion by gvsteve24** [bcaitech1/p4-dkt-freshtomato#3] [General]
# 잡담_주말에 뭐했어?

* 열라 놀았음!
* 머신러닝 devmatching ! 

# 회의 기록

### 깃헙!

* 깃헙을 보고 왔는데 어렵진 않았다.
* ZenHub 등 다른 탭들을 이용한 것으로 보인다.
* 위키 - 팀내에서 남기는 개인 일지
  * 위키에서는 진행사항을 이야기 했지만 큰 부분에서 실험내용을 파악하기 힘들었다.
  * 개인이 남긴 족적을 계속 중간에 파악할 수 있었다.
* 팀 Notion을 썼을 때 칸반보드를 썼는데 깃헙에서 *프로젝트*랑 유사함
  * zenhub이 프로젝트를 좀 더 세련되게 만든 걸로 보임
* 디스커션 탭이 인상적이었음.
  * Idea, QnA, Announcement등이 있어서 활용도가 좋을것 같음
* 피어세션 일지는 어디에 적는게 좋을까?
  * 위키에 개인적으로 일지를 남기는 걸로 하자.
* 프로젝트 실험 관련해서는 프로젝트 탭에 올리기
* 이슈는 개발 관련해서 생성되는 토론
  * 코드 리팩토링, 기능개발, 버그 픽스 등을 씀
* 실험들 레포랑 나중에 모델 서빙할 때 레포를 따로 쓰자

* 어떤게 좋을까 ? Git Flow? Fresh Tomato's 브랜치 전략?

  * 거의 둘다 비슷함!
  * 차이점은 배포가 없고 / 각자 브랜치를 가져간다는 것

  

* 앞으로 진행해볼 만한 것들 공유

  (공통) 시드를 바꿔가면서 퍼블릭이랑 유사하게 나오는 것 찾기

  * 재훈
    * 시계열 데이터를 어떻게 다뤘는지 궁금하다
    * 게임데이터 컴피티션 피처 썼던 공유 (차원의 저주)
  * 민용
    * [정형데이터에서 시계열 피처 생성 기록](http://boostcamp.stages.ai/competitions/3/discussion/post/228)
    * [transformer 사용한 논문](https://arxiv.org/pdf/1506.05908.pdf) 읽어보고 진행해볼것 같다
    * [논문 목록 참고!](https://www.kaggle.com/c/riiid-test-answer-prediction/discussion/188911)
  * 아라
    * 베이스라인 읽어보고
    * [논문](https://paperswithcode.com/paper/a-self-attentive-model-for-knowledge-tracing#code) 읽고!
    * EDA해보고 
    * automl 돌려보고 있음
  * 유라
    * paperswithcode 참고해서 가져올 수 있는 모델 보려구
    * EDA, 데일리미션 해볼 생각

  

* 하이퍼파라미터는 크게 신경안쓰고 (automl 있으니) 실험하면 좋을 것 같다.

* Validation Set을 만드는게 어려웠다. 

  * Time nested CV ? 오히려 안나왔던 기억이... 
  * 사람별로 split을 해보는게 일리있는 방법 같다.

  

# 그라운드 룰

### 팀원

강민용 T1001,
김진현 T1248,
문재훈 T1058,
배아라 T1084,
이정현 T1160,
최유라 T1212,

### 피어세션시간

오후 4:00 ~ 

### Ground rules

- 오늘의 TMI + 오늘의 이모티콘 공유
- 끝인사 + 저녁 식사 공유

### Github를 활용한 협업

**discussion tab**

📢 [Announcements]
💬 [General]
🙏 [Q&A] 
🍯 [Tips]
✨ [Update]

**projects tab**

- issues 활용

**wiki tab**

- 개인 일지 작성

### Github team repo

[https://github.com/bcaitech1/p4-dkt-freshtomato](https://github.com/bcaitech1/p4-dkt-freshtomato)

# TMI _ 오늘먹은점심(+먹을저녁)

* (정현) 비빔밥 + **돈까스**
* (아라) 들깨수제비 + **갈비**
* (재훈) 스팸
* (민용) 김치볶음밥
* (진현) 라멘
* (유라) 돈까스

**Comment by arabae** [comment]
와... 감동입니다.. 어떻게 이렇게 정리를 잘하시죠 😲❗

**Comment by Yuuraa** [comment]
진심 최고입니다..!! 최고의 피어 일지.. 칭찬합니다... 👍👍👍👍
